convex/lib/rowLevelSecurity.ts

```diff
@@ -365,6 +391,16 @@ class WrapWriter<Ctx, DataModel extends GenericDataModel>
     }
     return await this.db.insert(table, value);
   }
+  tableName<TableName extends string>(
+    id: GenericId<TableName>
+  ): TableName | null {
+    for (const tableName of Object.keys(this.rules)) {
+      if (this.db.normalizeId(tableName, id)) {
+        return tableName as TableName;
+      }
+    }
```

Comment:
This makes me nervous to suggest this as a serious solution for folks looking for RLS. Between this and Zod, double checking: it's not possible to expose a `getTableName` non-db function only callable from the server @sshader?